www/source/docs/habitat-cli.html.md

```diff
@@ -1334,7 +1334,7 @@ hab pkg binlink [FLAGS] [OPTIONS] <PKG_IDENT> [BINARY]
 **OPTIONS**
 
 ```
--d, --dest <DEST_DIR>    Sets the destination directory [env: HAB_BINLINK_DIR=/hab/bin]  [default: /bin]
+-d, --dest <DEST_DIR>    Sets the destination directory [env: HAB_BINLINK_DIR=]  [default: /bin]
```

Comment:
Do we know why this changed?   Not a blocker, but I'm curious if this is something unique to the executing environment that generates the docs, or if there was a code change I missed.  

I suspect the former, and if so, what could we do to make this consistent across doc generation? (Automation is the correct answer :joy_cat: )